library/src/scala/compiletime/package.scala

```diff
@@ -117,13 +117,9 @@ transparent inline def constValue[T]: T =
  *  `(constValue[X1], ..., constValue[Xn])`.
  */
 inline def constValueTuple[T <: Tuple]: T =
```

Comment:
Should the same be done for `summonAll`, or do we need to keep runtime support for it?

Reply:
We should do the same in `summonAll`. I updated `summonAll`.

There is no runtime support for these methods, they are erased.